testplan/testing/multitest/result.py

```diff
@@ -1871,6 +1870,42 @@ def matplot(self, pyplot, width=2, height=2, description=None):
             description=description
         )
 
+    @bind_entry
+    def graph(self, graph_type, graph_data, description,
+              series_options, graph_options):
+        """
+        Displays a Graph in the report.
+
+         code-block:: python
```

Comment:
This doesn't render correctly for me. I think you need to add leading dots and make sure the code block is indented, like:

```
.. code-block:: python

    result.graph(...
```

There are plenty of examples in this same module you can copy!